pgm-2.txt

```diff
@@ -0,0 +1,7 @@
+
+#!/bin/bash
+echo "hellooooooo"
+find *.log > z.txt
+find *.txt >> z.txt
+sudo mkdir /home/sudhir/zout
+tar -zczf matches.tar.gz /home/main
```

Comment:
What if we a file called config.xml and values.dat under the /home/main directory, will they also get into the tar.
The idea was to create a selective tar of only the zip files and log files.